template/example/types.d.ts

```diff
@@ -0,0 +1,20 @@
+import {MMap} from '@mappable-world/mappable-types';
+
+declare global {
+    const React: typeof import('react');
+    const ReactDOM: typeof import('react-dom');
+    const Vue: typeof import('@vue/runtime-dom');
+    let map: MMap;
+
+    interface Window {
+        map: MMap;
+    }
+}
+
+declare module '@mappable-world/mappable-types/import' {
+    interface Import {
+        (pkg: '%PACKAGE_NAME%'): Promise<typeof import('../src/index')>;
+    }
+}
```

Comment:
this should be a part of generated package, not for example

Reply:
hello! I have created a new [pull request](https://github.com/mappable-world/mappable-cli/pull/9) for the proposed changes